hasjob/views/listing.py

```diff
@@ -686,7 +686,7 @@ def confirm(domain, hashid):
     # We get here if it's (a) POST_STATE.UNPUBLISHED and (b) the user is confirmed authorised
     if 'form.id' in request.form and form.validate_on_submit():
         # User has accepted terms of service. Now send email and/or wait for payment
-        msg = Message(subject="Confirmation of your job post at Hasjob",
+        msg = Message(subject="Confirm your job post at Hasjob - \"{headline}\"".format(headline=post.headline),
```

Comment:
Use `subject="Confirm your job post: \"{headline}\"".format(headline=post.headline)`

1. The sender is already Hasjob.
2. The employer may be using a sub-board not named Hasjob.